apps/src/templates/sectionProgressV2/SectionProgressSelector.jsx

```diff
@@ -27,7 +28,25 @@ function SectionProgressSelector({
   sectionId,
   hasSeenProgressTableInvite,
 }) {
-  const [hasJustToggledViews, setHasJustToggledViews] = React.useState(false);
+  const [hasJustToggledViews, setHasJustToggledViews] = useState(false);
+
+  useEffect(() => {
+    const params = queryParams('view');
+    if (params === 'v2') {
+      setShowProgressTableV2(true);
+      setHasJustToggledViews(true);
+      new UserPreferences().setShowProgressTableV2(true);
```

Comment:
@dju90 this should be resetting the UserPreferences.  I should have commented on the thread that this was included.